### ISSUE TITLE
If height is set in overlayCSS, height still set to 100%

### ISSUE THREAD

**Issue by jpcody** [buckwilson/Lightbox_me#13]
This could be user error, but when I pass a fixed height to overlayCSS (600px in my case), my $(window).height() < $(document).height() evaluates false, and after setting the CSS, it is reset to 100%. I fixed it by updating line 160 to the following:

```
if(opts.overlayCSS.hasOwnProperty('height') == false){
    $overlay.css({height: '100%'});
}
```

I'd love to hear if this was an error on my part.

**Comment by buckwilson** [comment]
Yeah the way it works is it won't let you override the height or width of the overlay. I'm not sure if this is a feature or a bug myself...

What's the use case where you wanted to guarantee the overlay be a certain height?
